app/router/student.py

```diff
@@ -125,3 +130,79 @@ async def verify_student(request: Request, student_id: str):
             return HTTPException(status_code=404, detail="Student ID does not exist!")
         return True
     return HTTPException(status_code=response.status_code, detail=response.errors)
+
+
+def get_jnv_code(data):
+    response = requests.get(
+        school_db_url, params={"region": data.region, "name": data.school_name}
+    )
+    if response.status_code == 200:
+        if len(response.json()) == 0:
+            return HTTPException(
+                status_code=404, detail="JNV or region does not exist!"
+            )
+        return response.json()
+    return HTTPException(status_code=response.status_code, detail=response.errors)
+
+
+def generate_three_digit_code(code):
+    for _ in range(3):
+        code += random.randint(0, 9)
```

Comment:
`code += str(random.randint(0, 9))`

Also write an example in function comments maybe? 
`"""if input code = '234', output = '234910'"""`